src/Symfony/Component/Serializer/Attribute/SerializedName.php

```diff
@@ -21,9 +21,10 @@
  * @Target({"PROPERTY", "METHOD"})
  *
  * @author Fabien Bourigault <bourigaultfabien@gmail.com>
+ * @todo
```

Comment:
?

Reply:
Oops, started to type a todo to restore the `final` before I concluded there was no reason to. Fixed  😆